apps/api-documenter/src/cli/GenerateAction.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+
+import { ApiDocumenterCommandLine } from './ApiDocumenterCommandLine';
+import { BaseAction } from './BaseAction';
+
+import { ExperimentYamlDocumenter } from '../documenters/ExperimentYamlDocumenter';
+import { ApiModel } from '@microsoft/api-extractor-model';
+import { DocumenterConfig } from '../documenters/DocumenterConfig';
+import { FileSystem } from '@microsoft/node-core-library';
+import { IConfigFile } from '../documenters/IConfigFile';
+
+export class GenerateAction extends BaseAction {
+  constructor(parser: ApiDocumenterCommandLine) {
+    super({
+      actionName: 'generate',
```

Comment:
>generate [](start = 19, length = 8)

This seems like a weird verb. Maybe "experimental-yaml" or something?

Reply:
See [this comment](https://github.com/microsoft/web-build-tools/pull/1265#discussion_r284569163). We discussed it with @octogonz that we might wanna move to a using this generic `generate` action along with a config file that will command which documenter to be used.